db/models.py

```diff
@@ -1 +1,50 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+    def __str__(self) -> str:
+        return self.name
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(
+        Race,
+        on_delete=models.CASCADE,
+    )
```

Comment:
don't forget to add related_name for ForeignKey and ManyToMany fields

Reply:
Why do I need to set it if the tests expect it to be default?